src/shared-components/button/components/roundButton/style.js

```diff
@@ -62,6 +64,54 @@ export const roundButtonLoader = disabled => css`
   `};
 `;
 
-export const RoundButtonText = styled.p`
+export const RoundButtonText = (color, textColor) => css`
+  color: ${buttonTextColor(color, textColor)};
   margin: 10px 0;
 `;
+
+/**
+ * Given a color as an argument,
+ * determine an alternate color for pairing
+ * @param  string color   the current color name of the round button (e.g purple, primary, etc.)
+ * @return string         hex string of the alternate color (e.g. #efefef)
+ */
+const determineAlternateTextColor = color => {
```

Comment:
i didnt put this as a util as its kind of opinionated as to whats a derived alternate color. This works great for the round button color vs text color, but this may not be the formula we want across the board, so i left this as helper functions in the roundbutton styles specifically. Happy to break this out if we need.